torch/_inductor/cpu_vec_isa.py

```diff
@@ -306,9 +320,22 @@ def _check_and_append_supported_isa(
 
     return supported_isa
 
+def _is_arm_neoverse_v1() -> bool:
+    # reference: https://github.com/ARM-software/ComputeLibrary/blob/main/src/common/cpuinfo/CpuModel.cpp
+    
+    with open("/proc/cpuinfo") as _cpuinfo:
+        while True:
```

Comment:
I think it'd be better practice to avoid `while True` loops and avoid infinite loops and instead do 
```
line = _cpuinfo.readline()
while line:
```